client/src/index.js

```diff
@@ -12,6 +12,10 @@ import sagas from './sagas';
 
 import * as serviceWorker from './serviceWorker';
 
+import ReactGA from 'react-ga';
+const TRACKING_ID = "UA-129523795-1";
```

Comment:
let's take this out. We will place this in the .env so reference here with process.env.GA_TRACKING_ID

Reply:
Good thought, thank you! Just fixed.